build.sbt

```diff
@@ -53,6 +55,7 @@ lazy val root = (project in file("."))
 lazy val scalaClasses = (project in file("scala"))
   .settings(commonSettings)
   .settings(
+    publishArtifact := false,
```

Comment:
Just checking that publishing is disabled intentionally here for production merge?

Reply:
Good catch that was there for a test. I've removed it